packages/sdk/react-client/src/testing/ClientRepeater.tsx

```diff
@@ -21,41 +25,102 @@ export type ClientRepeaterProps<P extends RepeatedComponentProps> = {
   registerSignalFactory?: boolean;
   className?: string;
   Component: FC<any>;
+  types?: TypeCollection;
+  networkToggle?: boolean;
+  createIdentity?: boolean;
+  createSpace?: boolean;
+  onCreateSpace?: (space: Space) => MaybePromise<void>;
   args?: Omit<P, 'id' | 'count'>;
 };
 
 /**
  * Utility component for Storybook stories which sets up clients for n peers.
  * The `Component` property is rendered n times, once for each peer.
  */
-// TODO(burdon): Reconcile with ClientSpaceDecorator.
+// TODO(wittjosiah): Rename.
 export const ClientRepeater = <P extends RepeatedComponentProps>(props: ClientRepeaterProps<P>) => {
-  const { clients, count = 1, className = 'flex w-full place-content-evenly', Component } = props;
+  const {
+    count = 1,
+    className = 'flex w-full place-content-evenly',
```

Comment:
If this is getting passed to `mx` the prop name should be `classNames` and the type should be `ClassNameValue`, also available as `ThemedClassName<E extends HTMLElement>`.

Reply:
I think its only being passed to the div on line 90